front/src/Components/Chat/Chat.svelte

```diff
@@ -16,6 +23,12 @@
         if (autoscroll) listDom.scrollTo(0, listDom.scrollHeight);
     });
 
+    function onClick(e: MouseEvent) {
+        if (!e.composedPath().find((et) => et === chatWindowElement)) {
```

Comment:
Maybe put this condition into an Htmlutils function: isClickedOutside(event, target)